indy_common/constants.py

```diff
@@ -160,6 +171,10 @@
 GET_REVOC_REG_DELTA = IndyTransactions.GET_REVOC_REG_DELTA.value
 CHANGE_KEY = IndyTransactions.CHANGE_KEY.value
 
+# Rich Schema
+SET_CONTEXT = IndyTransactions.SET_CONTEXT.value
```

Comment:
Please extend `https://github.com/hyperledger/indy-node/blob/master/docs/source/transactions.md` and `https://github.com/hyperledger/indy-node/blob/master/docs/source/requests.md` with a description of new requests (SET_CONTEXT and GET_CONTEXT) and transaction (SET_CONTEXT)

Reply:
added in [20ea2b2](https://github.com/hyperledger/indy-node/pull/1430/commits/20ea2b297ba8969fb8e9cc183e678ca2dbf5411c)